README.md

```diff
@@ -69,6 +87,26 @@ console.log(`Value: ${values.toString()}`)
 await listener.close()
 ```
 
+### Custom filters Example
+
+You can create your own address filters for this transports, or rely in the filters [provided](./src/filters.js).
+
+```js
+const WS = require('libp2p-websockets')
+const filters = require('libp2p-websockets/src/filters')
+
+const ws = new WS({ upgrader, filter: filters.all })
```

Comment:
Yeah I think the libp2p transport configuration example will be ideal here, https://github.com/libp2p/js-libp2p/blob/master/doc/CONFIGURATION.md#customizing-transports.

Reply:
I added the libp2p usage example, but kept the other. They could still be relevant, but the Readme might get too noisy. What's your opinion?